spec/unit/htty/cli/commands/path_set_spec.rb

```diff
@@ -67,6 +67,34 @@
         built = klass.build_for('x baz', :session => :another_session)
         built.should == nil
       end
+
+      it 'should correctly URL escape simple paths' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.